packages/core/test/call.ts

```diff
@@ -762,14 +762,61 @@ m.call(..., { id: "MyUniqueId"})`
         (v) => v.type === FutureType.CONTRACT_CALL
       );
 
-      await assert.isFulfilled(
```

Comment:
`assert.isFulfilled` would succeed even if the promise errored (as technically the promise is complete). I switched to getting the result and asserting that there were no validation failures.

Reply:
You are correct I mispoke. The issue is that the underlying validation could fail and the test could still pass.
Switching to asserting on the return result allowed the test to fail in the presence of unexpected validation failures.